rpcs3/rpcs3qt/pkg_install_dialog.cpp

```diff
@@ -123,19 +137,32 @@ pkg_install_dialog::pkg_install_dialog(const QStringList& paths, game_compatibil
 		}
 	});
 
-	connect(m_dir_list, &QListWidget::itemChanged, this, [this, buttons](QListWidgetItem*)
+	connect(m_dir_list, &QListWidget::itemChanged, this, [this, installation_info, buttons](QListWidgetItem*)
 	{
-		bool any_checked = false;
+		u64 data_size = 0;
+		u64 free_space = 0;
+
+		// Retrieve disk space info on data path's drive
+		if (fs::device_stat stat{}; fs::statfs(rpcs3::utils::get_hdd0_game_dir(), stat))
```

Comment:
I think this should only be done once when the dialog opens

Reply:
originally I did that, then I moved it here because it is not time consuming and allows to dynamically update the free space in case users makes some clean up on the target disk